src/config.json

```diff
@@ -20,7 +20,9 @@
         "extension_url": "https://s3.amazonaws.com/extend.brackets/{0}/{0}-{1}.zip",
         "linting.enabled_by_default": true,
         "build_timestamp": "",
-        "healthDataServerURL": "https://healthdev.brackets.io/healthDataLog"
+        "healthDataServerURL": "https://health.brackets.io/healthDataLog",
+        "analyticsDataServerURL" : "https://cc-api-data-stage.adobe.io/ingest/",
```

Comment:
Looks like this is still referring to stage server. I think you need to create a grunt task to point to the actual server. Also don't change `healthDataServerURL` from stage to prod as the grunt job takes care of it and the `analyticsDataServerURL` should be doing the same thing.

Reply:
I have added the stage and prod URL and is available in current commit